app/controllers/landmarks_controller.rb

```diff
@@ -33,9 +33,20 @@ def index
       nearby_results = JSON.parse(nearby_response.body, symbolize_names: true)[:results]
 
       # Return an array of landmark objects
+      require "pry"; binding.pry
       @landmarks = nearby_results.map do |landmark_data|
         Landmark.new(landmark_data)
       end
+      @parks = @landmarks.select{|landmark| landmark.types == ["park recreation area"]}
+      @museums = @landmarks.select{|landmark| landmark.types == ["museum"]}
+      @theaters = @landmarks.select{|landmark| landmark.types == ["theater"]}
```

Comment:
We call it "Theatre", but ok.

Reply:
Unless theatre is used in the proper name of a building, production company, etc., theater is the preferred spelling in American English